spec/input/manifest.spec.yml

```diff
@@ -69,6 +69,8 @@ spec:
               - Collect logs and metrics from Apache instances
           deployment_modes:
             $ref: "../integration/manifest.spec.yml#/definitions/deployment_modes"
+          configuration_links:
```

Comment:
Nit. What is the reason to mark these as `configuration_links`? Is it because they are expected to appear around configuration steps? `links` could be used in the future for other things, like links to additional documentation and so on.

Reply:
Yes there was another comments saying that `links` was too generic, but if you think it's better to leave it generic on purpose I can put it back.

Reply:
Oh ok, looks good then, not a strong opinion from my side.